schema/methods_directory/local-orbital.json

```diff
@@ -15,9 +15,35 @@
         },
         "subtype": {
             "enum": [
-                "gto",
-                "sto",
-                "cgto"
+                "pople",
+                "dunning"
+            ]
+        },
+        "polarization": {
+            "enum": [
+                "none",
+                "heavy_atoms",
+                "all_atoms"
+             ]
+        },
+        "diffuse": {
+            "enum": [
+                "none",
+                "heavy_atoms",
+                "all_atoms"
+             ]
+         },
+        "basis": {
```

Comment:
1. For the sake of simplicity, I'd suggest omitting the rest and only keeping the `basisSet` (instead of `basis`) as the key. There is also a way to reuse a `slugified_entry` object as more flexible, probably, as sometimes we would want to omit having to include `the inner structure `{"basisSet": {"slug": "xxx"}}` vs just `{"basisSet": "xxx"}`
2. JSON schema supports the "description" field, which can be handy